catboost/libs/data/target.cpp

```diff
@@ -417,6 +417,24 @@ ERawTargetType TRawTargetDataProvider::GetTargetType() const {
     return Data.TargetType;
 }
 
+
+bool TRawTargetDataProvider::IsTargetsContainsNan() const {
```

Comment:
it is not required anymore

Reply:
True! Now is much more elegant!
Thank you @LyzhinIvan 
Done.